### ISSUE TITLE
Create a File > Export... submenu

### ISSUE THREAD

**Issue by dacap** [aseprite/aseprite#1252]
There are several features that could be grouped in just one submenu.

* Export one frame (#1080)
* Export animation as a GIF (same as `File > Save Copy As...` with `.gif` extension)
* Publish to social network (#1220)
* Export Sprite Sheet
etc.

**Comment by Ashedragon** [comment]
Does this include the ability to export a GIF animation of specific tags, in the same way exporting a sprite sheet functions?

**Comment by dacap** [comment]
Yes, some options from the File > Export menu should use #1080, see this comment https://github.com/aseprite/aseprite/issues/1080#issuecomment-244763783

**Comment by littletinman** [comment]
I would very much like the ability to export a marquee selection. This is a make or break feature for me.

**Comment by oceanhahn** [comment]
The workaround I use for that right now is either to Copy Merged to a new file and export that, or use New Sprite From Selection and save that (which transfers colour data like palette and mode state but neglects layers other than current).

Being able to save a selection directly to a new file would be nice, though, especially if it could save through layers and frames! :D
